assignment5.txt

```diff
@@ -0,0 +1,10 @@
+Jack Rosen
+1. int main() {
+	int a = 0, c = 0;
+	float b;
+	fxn(a,b,c);
+	return 0;
+}
+My code would look like this. I would instantiate the variables and then call the function.
+2. An iteration uses loops to continue going but a recursion calls itself to keep it going. There are times where iteration can take longer and times where recursion can use up more storage. Recursion can be worst because if it is endless because it keeps taking up more space.
```

Comment:
right, complexity is key! and now that we're learning about memory management, you can further understand this by understanding that each time a function is called, a stack is initialized. So having a recursive function can be very memory ineffective.